include/exadg/acoustic_conservation_equations/spatial_discretization/spatial_operator.cpp

```diff
@@ -331,7 +331,9 @@ void
 SpatialOperator<dim, Number>::apply_inverse_mass_operator(BlockVectorType &       dst,
```

Comment:
This function name is now unclear in my opinion. `apply_inverse_mass_operator` should really be just the inverse mass. I could imagine we call this then `inverse_time_derivative_operator` (or `scaled_inverse_mass_operator`, with a documentation of what `scaled` means exactly in the context of two variables, i.e. velocity and pressure).

Reply:
[b7e5645](https://github.com/exadg/exadg/pull/631/commits/b7e564573a2e19ac1a69a3dc9ef2baab1755b1a4)